libreosteoweb/api/invoicing/generator.py

```diff
@@ -59,7 +59,7 @@ def generate_invoice(self, examination, serializer_data, request):
         if self.therapeut_settings.invoice_footer is not None :
             invoice.footer = self.therapeut_settings.invoice_footer
         invoice.number = self.get_invoice_number()
-        invoice.date = datetime.today()
+        invoice.date = timezone.now() 
```

Comment:
En lisant le patch, c'est surprenant étant donné que datetime.today() donne un objet de type `date` (qui n'intègre jamais la notion de timezone, donc qui ne devrait pas poser de souci) et `timezone.now()` un objet de type `datetime` (qui peut intégrer la notion de timezone).

Reply:
C'est en effet l'origine des messages du type : 
`RuntimeWarning: DateTimeField Examination.date received a naive datetime (2019-07-01 00:00:00) while time zone support is active.
  RuntimeWarning)`
Le fait que Django considère un timezone, alors que la date passée dans l'objet n'a pas la notion de Timezone.